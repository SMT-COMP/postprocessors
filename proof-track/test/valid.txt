0.00/0.00	; run solver
1.51/0.67	; checking proof
2.49/1.12	valid
2.49/1.15	; done
2.49/1.15	EOF
