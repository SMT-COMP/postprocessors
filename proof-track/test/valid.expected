starexec-result=unsat
reason=valid
